phoenix-core/src/main/java/org/apache/phoenix/index/GlobalIndexChecker.java

```diff
@@ -93,10 +106,11 @@
  * the verified version that is masked by the unverified version(s).
  *
  */
-public class GlobalIndexChecker extends BaseRegionObserver {
+public class GlobalIndexChecker extends CompatGlobalIndexChecker {
```

Comment:
We don't need CompatGlobalIndexChecker now that we don't annotate indexes so I removed it